liquidctl/driver/krakenz3.py

```diff
@@ -0,0 +1,676 @@
+"""liquidctl drivers for fourth-generation NZXT Kraken X and Z liquid coolers.
+
+Supported devices:
+
+- NZXT Kraken X (X53, X63 and Z73)
+- NZXT Kraken Z (Z53, Z63 and Z73)
+
+Copyright (C) 2020–2022  Tom Frey, Jonas Malaco, Shady Nawara and contributors
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+# uses the psf/black style
+
+import itertools
+import io
+import math
+import logging
+
+try:
+    # The hidapi package, depending on how it's compiled, exposes one or two
+    # top level modules: hid and, optionally, hidraw.  When both are available,
+    # hid will be a libusb-based fallback implementation, and we prefer hidraw.
+    import hidraw as hid
+except ModuleNotFoundError:
+    import hid
```

Comment:
What is the reasoning for using hidapi directly, instead of through our `HidapiDevice` wrapper (in `liquidctl.usb`)?